public/assets/application-1bbe3d720eccb0f7b0efcf81d99a584fe409741d8465837f1dd6c1a946f16ecf.css

```diff
@@ -1,6 +1,7 @@
 /* Materialize Styling */
 @import "https://fonts.googleapis.com/icon?family=Material+Icons";
 @import url("https://fonts.googleapis.com/css?family=Crafty+Girls|Bentham|Ramabhadra");
+@import url("https://fonts.googleapis.com/css?family=Berkshire+Swash|Cabin+Sketch|Fredericka+the+Great|Handlee|Kranky|Patrick+Hand+SC|Righteous|Walter+Turncoat");
```

Comment:
I'm not sure what this file is.... Do we need to update this import for the fonts or will it update automatically?